frontend/src/components/ExperimentCollection/ExperimentCollection.js

```diff
@@ -6,10 +6,9 @@ import Loading  from "../Loading/Loading";
 const ExperimentCollection = ({match}) => {
     const [experimentCollection, loadingExperimentCollection] = useExperimentCollection(match.params.slug);
     const dashboard = experimentCollection?.dashboard;
-    const experiment = dashboard ? undefined : experimentCollection;;
+    const experiment = dashboard ? undefined : experimentCollection;
```

Comment:
It's only the removal of the semicolon, but what does this expression actually do? 😅 If dashboard is truthy then experiment should be undefined and otherwise it should be a collection? 🤔

Reply:
I realized that this is a bit of overkill anyway. Added a docstring explaining the possible structures of the backend response, and refactored such that reasoning is over `experimentCollection` (i.e., the object resulting from that response), instead of derived objects.